tests/pipeline/test_events.py

```diff
@@ -456,3 +460,53 @@ def make_loader(cls, events, next_value_columns, previous_value_columns):
             next_value_columns,
             previous_value_columns,
         )
+
+
+class EventLoaderUtilsTestCase(ZiplineTestCase):
+    dates = pd.to_datetime([
+        '2013-01-04 3:00:00',
```

Comment:
I think we should make these cases closer to the boundary conditions, e.g.:

`pd.Timestamp('2013-01-04 8:44:59', tz='US/Eastern').tz_convert('UTC')`
`pd.Timestamp('2013-01-04 8:45:00, tz='US/Eastern').tz_convert('UTC')`